.travis.yml

```diff
@@ -56,3 +56,10 @@ jobs:
     - stage: Lint
       install: travis_retry composer require --dev --prefer-dist phpstan/phpstan:^0.7
       script: vendor/bin/phpstan analyse -l 3 -c phpstan.neon lib tests
+
+    - stage: Benchmark
+      install:
+        - travis_retry composer update --prefer-dist
+        - curl -o phpbench.phar https://phpbench.github.io/phpbench/phpbench.phar
```

Comment:
I know I'm a weirdo, but I'd rather `composer require` with a specific version here. I don't consider `curl` to be a good way to install things...

Reply:
I'd too, but it's not possible, as said above:

> It's not possible to have phpbench/phpbench as dev-dependency due to back-reference to doctrine/annotations 1.x.

Also `curl`ing specific version is currently not possible as versioned PHARs are not published (already discussed with @dantleech to be possible in future).

Reply:
Otherwise a phar could be downloaded with phive, but I had a few issues with travis in the past.

Reply:
Still same issues: would need to be in the dependency graph. Let's tackle the problem once we hit 2.0.0 (stable) then, and a dependency loop *can* be re-introduced, if needed.